### PR TITLE
QuickStart - Public Cloud

### PR DESCRIPTION
**Pull request by Loquacity**
Rewriting begins here.

### CODE REVIEW

**Comment by Loquacity** [comment]
[suse_manager_quickstart-public-cloud_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/4384930/suse_manager_quickstart-public-cloud_guide.pdf)

File: modules/quickstart-public-cloud/pages/installation.adoc [hunk 1]
```diff
@@ -1,22 +1,51 @@
...
-// Check this. LKB 2020-03-06
 For more information on activating public clouds, see xref:installation:pubcloud-setup.adoc[].
 
-A {productname} administrator account is created by default.
-You need this account to access configuration settings in the public cloud provider interface.
+
+Log in to the {productname} {webui} with a web browser, using an address like this:
+
+----
+https://<public_IP>
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/installation.adoc]: actually by IP?  I think we'd better say "by the name that you'll see at the end of the setup routine.  We have this in our default install instruction.

File: modules/quickstart-public-cloud/pages/installation.adoc [hunk 2]
```diff
@@ -1,22 +1,51 @@
...
+----
+
+Log in to the {productname} {webui} with the administrator account.
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/installation.adoc]:
```suggestion
Log in to the {productname} {webui} with the administrator account.
```
```suggestion
Sign in to the {productname} {webui} with the administrator account.
```

File: modules/quickstart-public-cloud/pages/installation.adoc [hunk 3]
```diff
@@ -33,47 +62,27 @@ The username and password varies depending on your provider.
 When you have logged in to the administrator account for the first time, change the default password to protect your account.
 ====
 
+For more information about setting up your {productname} Server, see xref:installation:server-setup.adoc[].
 
-Add the required software products to your client, and schedule a repository synchronization.
 
+Use the {productname} {webui} to add the required software products, and schedule a repository synchronization.
+The simplest way to do this is to navigate to menu:Admin[Setup Wizard] and follow the prompts.
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/installation.adoc]:
```suggestion
The way to do this is to navigate to menu:Admin[Setup Wizard] and follow the prompts.
```

File: modules/quickstart-public-cloud/pages/installation.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/quickstart-public-cloud/pages/installation.adoc]: Yeah, so the IP is much easier to find in the public cloud interface. I don't recall seeing an actual URL to use, I'll need to run through it again.

**Comment by keichwa** [comment]
On 2020-03-27 02:32, Lana Brindley wrote:

> +https://<public_IP>
> 
> Yeah, so the IP is much easier to find in the public cloud interface.

This PC world is tricky.

-- 
Karl Eichwalder

**Comment by Loquacity** [comment]
I'm going to merge this one, and will start a new PR when we get some feedback.

**Pull request merged**